docs/en/docs/advanced/extending-openapi.md

```diff
@@ -248,3 +248,27 @@ Now, to be able to test that everything works, create a *path operation*:
 Now, you should be able to disconnect your WiFi, go to your docs at <a href="http://127.0.0.1:8000/docs" class="external-link" target="_blank">http://127.0.0.1:8000/docs</a>, and reload the page.
 
 And even without Internet, you would be able to see the docs for your API and interact with it.
+
+
+### Add Tags
+
+You can group path by tags in both docs and redoc by using tags, tags also allows you to put descriptions and links. Tags description suppports markdown.
```

Comment:
We already have docs about tags here: https://fastapi.tiangolo.com/tutorial/path-operation-configuration/?h=tags#tags and here https://fastapi.tiangolo.com/tutorial/metadata/?h=tags#use-your-tags
So don't see a use for this?

Reply:
Well I must have missed the second one, and the first one uses a slightly different way of defining tags; 
Seeing this, I think the part about Tags sorting could be added to https://fastapi.tiangolo.com/tutorial/metadata/?h=tags#order-of-tags which is immediately after the second link you gave.

Also in the extending openapi doc we need to have tags defined so maybe we can just skip the Add Tags and instead put a link toward the doc here https://fastapi.tiangolo.com/tutorial/metadata/#metadata-for-tags

Let me know what you think so I can rewrite this part of the doc.

Reply:
I like adding it to https://fastapi.tiangolo.com/tutorial/metadata/?h=tags#order-of-tags, the https://fastapi.tiangolo.com/tutorial/metadata/#metadata-for-tags seems like a different thing?

Reply:
Hi, I included in the doc (metadata/order of tags) a link to the extending-openapi doc. metadata for tags is a bit different as it is the customization of the tags headers with markdown description and url link